source/includes/artifacts/_025-get-store-config-metadata.md.erb

```diff
@@ -0,0 +1,40 @@
+## Get Store Config Metadata
+
+This is a message that the plugin should, to allow users to configure global artifact stores from the Artifact Stores View in GoCD.
```

Comment:
`This is a message that the plugin should` -> `This is a message that the plugin should implement`